virttest/libvirt_vm.py

```diff
@@ -2456,6 +2456,56 @@ def get_interface_mac(self, interface):
         session.close()
         return mac.strip()
 
+    def get_network_cfg_file(self, iface_name):
+        """
+        Get absolute network cfg file path of the vm instance.
+
+        :param iface_name: Name of the network interface
+        """
+        session = self.wait_for_login()
+        vm_distro = self.get_distro()
+        iface_cfg_file = ""
+        if "ubuntu" in vm_distro.lower():
+            iface_cfg_file = "/etc/network/interfaces"
+        elif "suse" in vm_distro.lower():
+            iface_cfg_file = "/etc/sysconfig/network/ifcfg-%s" % (iface_name)
+        else:
+            iface_cfg_file = "/etc/sysconfig/network-scripts/"
+            iface_cfg_file += "ifcfg-%s" % (iface_name)
+        return iface_cfg_file
+
+    def set_network_param(self, iface_name, mac_addr, boot_proto, ip_addr,
+                          net_mask, **dargs):
+        """
+        Set network cfg parameter of the vm instance and returns the list
+
+        :param iface_name: Name of the network interface
+        :param mac_addr: Mac address of the network interface
+        :param boot_proto: static ip or dhcp of the interface
+        :param ip_addr: ip address for the interface
+        :param net_mask: network mask for the interface
+        :param dargs: dict of additional attributes
```

Comment:
@will-Do this API can used to configure and test guest with different n/w configurations and to persist after reboot, destroy and boot, migration etc.,
1. List can be dumped to network script file with \n as linesep
2. sure, I have changed the name as `create_vm_network_script`